### DISCUSSION TITLE
Build failds

### DISCUSSION THREAD

**Discussion by white5354** [eclipse-theia/theia#14086] [Q&A] [answered]
**Bug Description:** 
On windows 10, the build fails when doing yarn
**Steps to Reproduces:**
1. git clone https://github.com/eclipse-theia/theia
2. cd theia
3. yarn install
`node-gyp install
system cannot find the path specified。`
**Additional Information**
node v20.8.1
yarn 1.22.22
python 3.12.5

**Comment by msujew** [comment]
Hey @white5354,

are you cloning Theia into a directory that contains white space? node-gyp unfortunately cannot deal with whitespace in paths.

**Reply by white5354** [reply]
I did not

**Reply by msujew** [reply]
Any other specific info about your setup? Special characters in your path or something? node-gyp can be pretty finicky when it comes to a lot of things.

Without more context, there's little we can help with.

**Reply by white5354** [reply, marked as answer]
@msujew I am stupid, i can't npm install -g node-gyp. forget it. thanks for you
